figureqa/generation/data_utils.py

```diff
@@ -3,6 +3,7 @@
 
 import copy
 import random
+import numpy as np
```

Comment:
Thanks! Since we're checking for style, how about adding a newline before this import as per: https://www.python.org/dev/peps/pep-0008/#imports